### PR TITLE
Partridge claim for which year

### PR DESCRIPTION
**Pull request by lmcewanhmrc**
Alternative approach to legend on claim-for-which-year page

### CODE REVIEW

File: app/views/returns/credits/ClaimForWhichYearView.scala.html [hunk 1]
```diff
@@ -33,23 +35,27 @@
...
-        @paragraphBody(messages("claim-for-which-year.credits.paragraph.1"))
-
-        @govukRadios(
-            RadiosViewModel.apply(
-                field = form("value"),
-                legend = LegendViewModel(HtmlContent(hiddenContent("claim-for-which-year.credits.title-heading"))),
-                items = Seq(
-                    RadioItem(id = Some("value"), value = Some("true"), content =  Text("1 April 2022 to 31 March 2023"))
-                )
+        @if(form.errors.nonEmpty) {
```
Comment (mauro-fanton) [inline app/views/returns/credits/ClaimForWhichYearView.scala.html:38]: Would this way include the header in the error message (which I think we do not want) when we have a validation error?

Comment (lmcewanhmrc) [reply app/views/returns/credits/ClaimForWhichYearView.scala.html:38]: Might do, yeah. Will need to test that.

**Review by muninderadavelly — APPROVED** [review]

File: app/views/returns/credits/ClaimForWhichYearView.scala.html [hunk 2]
```diff
@@ -33,23 +35,27 @@
...
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @govukRadios(Radios(
```
Comment (liampan) [inline app/views/returns/credits/ClaimForWhichYearView.scala.html:42]: curious does this need to not use the RadiosViewModel? I cant see a difference

**Comment by lmcewanhmrc** [comment]
Not needed - going with hidden leg end

**Pull request closed without merging**